systemjs/webpack-code-splits/src/set-public-path.js

```diff
@@ -0,0 +1,3 @@
+import { setPublicPath } from 'systemjs-webpack-interop'
```

Comment:
I'm open to changing this not to use systemjs-webpack-interop to show the raw usage.